### DISCUSSION TITLE
v0.13.1 / v0.13.2

### DISCUSSION THREAD

**Discussion by ledermann** [solectrus/solectrus#2408] [Updates]
## Changes

- Status: Renders green color when status code is ok-ish, orange color otherwise (requires SENEC-Collector [v0.11.0](https://github.com/solectrus/senec-collector/releases/tag/v0.11.0) or later)
- Status: Changed wording from "LIVE/FAIL" to "CONNECTED/DISCONNECTED"
- Indicate when feed-in limitation is active (requires SENEC-Collector [v0.11.0](https://github.com/solectrus/senec-collector/releases/tag/v0.11.0) or later)
- Admin login: Added hint where to find the password in the `.env` file
- Registration banner: Added login button when Admin is not currently logged in
- ENV var `INFLUX_POLL_INTERVAL` can be used to set the InfluxDB polling interval (in seconds). Default is `5`
- ENV var `TZ` can be used to set the server timezone. Default is `Europe/Berlin`
- ENV var `FORCE_SSL` is by default set to `false`

## Fixes

- Fix adding of new points to the chart when server time is wrong (#2381)
- Fixes missing autarky circle when there is no consumption (#2319)
- Doesn't display outdated data when connection is lost

## Internal

- Code cleanup and refactoring
- Updated dependencies: Alpine Linux 3.18.4, Rails 7.0.8, Puma 6.4.0, Chart.js 4.4.0 (and lots more)

<hr /><em>This discussion was created from the release <a href='https://github.com/solectrus/solectrus/releases/tag/v0.13.1'>v0.13.1</a>.</em>

**Comment by StevieC121176** [comment]
Da ich keinen Sennen habe bekomme ich die Daten vom MQTT Collector. Mit der Version 13.1 bekomme ich nun ein "!" hinter dem Einspeisewert. Im Tooltip steht Begrenzung 0%. Ich habe generell keine 70% Regelung. Muss ich da noch irgendetwas in der `.env` ändern?
![IMG_1662](https://github.com/solectrus/solectrus/assets/113791299/37dcac5c-1375-4d4d-8b40-01aa9f2a81e7)
